dbconstraint/src/main/java/org/evomaster/dbconstraint/parser/jsql/JSqlConditionParser.java

```diff
@@ -3,21 +3,49 @@
 import net.sf.jsqlparser.JSQLParserException;
 import net.sf.jsqlparser.expression.Expression;
 import net.sf.jsqlparser.parser.CCJSqlParserUtil;
+import org.evomaster.dbconstraint.ast.SqlComparisonCondition;
+import org.evomaster.dbconstraint.ast.SqlComparisonOperator;
 import org.evomaster.dbconstraint.ast.SqlCondition;
 import org.evomaster.dbconstraint.parser.SqlConditionParser;
 import org.evomaster.dbconstraint.parser.SqlConditionParserException;
 
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
 public class JSqlConditionParser extends SqlConditionParser {
 
+    public static final String FORMULA_EQUALS_FORMULA_PATTERN = "\\(\\s*\\(([^<]*)\\)\\s*=\\s*\\(([^<]*)\\)\\s*\\)";
+
```

Comment:
can you provide bit of explanation for this Regex? eg, what is for

Reply:
Sure.